release-notes/azure/v11.2.1.md

```diff
@@ -1,13 +1,16 @@
 ## :zap: Giant Swarm Release 11.2.1 for Azure :zap:
 
+### This release contains Kubernetes ```1.16.8``` and other changes that increase cluster resiliency against API Throttling (error 429). Additional improvements include:
+- **Failed provisioning of VMs does not cause 429 anymore.**  We have added an automation that will reimage the failed instance immediately, before it exhausts API rate limits.
+- **Freeze of upgrade if API rate limits are almost exhausted.** Upgrades are causing a movement in deployments, updating templates as well as nodes, triggering more calls to API. Upgraded will be freezed automatically when limits are almost reached and resumed after situation is back to normal.
+- **Respecting the ```Retry After``` header returned by Azure API,** because not doing so made clusters facing the 429 error get stuck, together with every other cluster in the same subscription. Now clusters are able to recover by themselves and do not affect other clusters in the subscription.
```

Comment:
```suggestion
- **Respecting the ```Retry After``` header returned by Azure API,** because not doing so made clusters face the 429 error and get stuck, together with every other cluster in the same subscription. Now clusters are able to recover by themselves and do not affect other clusters in the subscription.
```

Reply:
Oh my understanding was that clusters still can face 429 but won't get stuck on it, they will hit it once and wait before doing anything again, that's why I worded it like this.